Scripting/Bindings.cs

```diff
@@ -64,7 +65,10 @@ public static void Bind(string KeyName, string FunctionName)
 		BindingObject NewBind = new BindingObject(KeyName, FunctionName);
 		Nullable<ButtonList> ButtonValue = null; //Making it null by default prevents a compile warning further down
 		Nullable<DIRECTION> AxisDirection = null; //Making it null by default prevents a compile warning further down
+		Nullable<JoystickList> ControllerButtonValue = null; // Making a new variable for Controller buttons because 
 		int Scancode = 0;
+		
+		
```

Comment:
Double blanklines are for separating functions and classes (moving to triple blank lines for separating classes in the same file)